src/main/java/seedu/address/model/person/PersonComparatorUtil.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.model.person;
+
+/**
+ * Utility class with useful methods for writing person comparators
+ */
+public class PersonComparatorUtil {
+
+    /**
+     * Compares two persons based on whether or not they are favorited. The favorited person will be ordered first,
+     * otherwise if both are favorited or both are unfavorited, then they are considered equal.
```

Comment:
Took me a few seconds to digest, maybe I'm slow >< How about this explanation:
If both persons have the same favorite status (yes/no), they are considered equal.